src/screens/Forum/ForumHomeScreen.js

```diff
@@ -1,10 +1,57 @@
+/* eslint-disable no-console */
 import React from 'react';
-import { View, Text } from 'react-native';
-
-export default function ForumHomeScreen() {
-  return (
-    <View>
-      <Text>Not implemented</Text>
-    </View>
-  );
+import { View, ScrollView } from 'react-native';
+import firestore from '@react-native-firebase/firestore';
+import ForumPost from './ForumPost';
+
+export default class ForumHomeScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { posts: [] };
```

Comment:
Add a loading state that displays an `<ActivityIndicator />` when the data is being loaded. See twitterrer